src/lib/common/config.ts

```diff
@@ -476,6 +476,11 @@ export interface ResourceConfig {
    */
   suites: string[];
 
+  /**
+   * An optional path to a tsconfig for ts-node. This options can be used
```

Comment:
```suggestion
   * An optional path to a tsconfig for ts-node. This option can be used
```

Reply:
Fixed the typo and added a note about setting it to `false` to avoid registering ts-node at all